plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -327,4 +323,12 @@ public static Object deserializePartitionValue(Type type, String valueString, St
 
         return Collections.unmodifiableMap(partitionKeys);
     }
+
+    public static LocationProvider getLocationProvider(String tableLocation, Map<String, String> properties)
+    {
+        if (properties.containsKey(WRITE_LOCATION_PROVIDER_IMPL)) {
+            throw new TrinoException(NOT_SUPPORTED, "Custom Iceberg location provider implementation is not supported");
```

Comment:
Nit: maybe be more verbose in error message. It would be nice to quote the class name of location provider used for table. Maybe sth like:
```
Table XXX specifies YYY as a location provider. Writing to Iceberg tables with custom location provider is not supported.
```